FirebaseAdmin/FirebaseAdmin.IntegrationTests/FirebaseAuthTest.cs

```diff
@@ -324,8 +324,15 @@ public async Task ListUsers()
                     if (users.Contains(uid) && !listedUsers.Contains(uid))
                     {
                         listedUsers.Add(uid);
-                        Assert.NotNull(enumerator.Current.PasswordHash);
-                        Assert.NotNull(enumerator.Current.PasswordSalt);
+                        var errMsgTemplate = "Missing {0} field. A common cause would be "
```

Comment:
It feels more natural to me to just inline the error message (despite some slight duplication). However, that runs afoul of stylecop, so a `#pragma warning disable` would be required. So I just went with the errMsgTemplate variable as seen here.

Using the `#pragma`, this block would look like this:
```c#
#pragma warning disable SA1118 // ParameterMustNotSpanMultipleLines
AssertWithMessage.NotNull(
    enumerator.Current.PasswordHash,
    "Missing PasswordHash field. A common cause would be forgetting to " 
    + "add the 'Firebase Authentication Admin' permission. See " 
    + "instructions in CONTRIBUTING.md");
AssertWithMessage.NotNull(
    enumerator.Current.PasswordSalt,
    "Missing PasswordSalt field. A common cause would be forgetting to " 
    + "add the 'Firebase Authentication Admin' permission. See " 
    + "instructions in CONTRIBUTING.md");
#pragma warning restore SA1118 // ParameterMustNotSpanMultipleLines
```